deploy/docker/x86_64/Dockerfile

```diff
@@ -47,7 +80,9 @@ RUN sed -i s/systemd// /etc/nsswitch.conf
 
 COPY --from=build /usr/local /usr/local/
 COPY --from=build /ganesha-extra /
-COPY nfs-provisioner /nfs-provisioner
+
+ARG binary=bin/nfs-provisioner
+COPY ${binary} /nfs-provisioner
```

Comment:
This step fails for me when I have run `docker build .` at this point both on my x86_64 and on arm64.

For your convenience, the build stage had this content in it after building.

```
[root@2d8e5873fb25 local]# ls /usr/local/ 
bin  etc  games  include  lib  lib64  libexec  sbin  share  src  var

[root@2d8e5873fb25 local]# ls /usr/local/bin/
ganesha.nfsd

[root@2d8e5873fb25 local]# ls /nfs-ganesha/
CMakeCache.txt		 MainNFSD   SAL			 coverity      idmapper		     log		support
CMakeFiles		 Makefile   avl			 dbus	       include		     os			test
CPackConfig.cmake	 Protocols  cidr		 doc	       install_manifest.txt  scripts		tools
CPackSourceConfig.cmake  README.md  cmake_install.cmake  ganesha.nfsd  jenkins		     sm_notify.ganesha
FSAL			 RPCAL	    config_parsing	 hashtable     libntirpc	     src

[root@2d8e5873fb25 local]# ls /ganesha-extra/
etc

[root@2d8e5873fb25 local]# ls /ganesha-extra/etc/
dbus-1
```

Reply:
I understand this to be the previous situation as well. I simply don't yet understand the build system yet.